src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/MissingOverrideCheck.java

```diff
@@ -81,6 +108,24 @@
  *       value="true"/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * class Test {
+ *
+ *     &#47;** {&#64;inheritDoc} *&#47;
+ *     public void equals() { // violation, not be annotated with &#64;Override
```

Comment:
`should not be` here too

Reply:
`should be annotated with &#64;Override ` here too